frontend/src/main/pages/AdminAnnouncementsPage.js

```diff
@@ -24,6 +24,7 @@ export default function AdminAnnouncementsPage() {
     const commonsName = commonsPlus?.commons.name;
 
     return (
+      // Stryker disable all
```

Comment:
```suggestion

```

Reply:
Thank you for helping me remove this line! I have to check every comments one by one in order to prevent that I may miss some of them